src/System.Net.Http.WinHttpHandler/tests/UnitTests/FakeInterop.cs

```diff
@@ -388,6 +388,15 @@ private static bool CopyToBufferOrFailIfInsufficientBufferLength(string value, I
             return true;
         }
 
+        public static bool WinHttpQueryOption(
+            SafeWinHttpHandle handle,
+            uint option,
+            ref uint buffer,
+            ref uint bufferSize)
+        {
+            return true;
```

Comment:
I'm not suggesting you change the signature.  I'm asking, for example, do you need to add a line:
```C#
data = Interop.WinHttp.WINHTTP_PROTOCOL_FLAG_HTTP2;
```
Otherwise the line at https://github.com/dotnet/corefx/pull/10948/files/3683316544a2b59269d66ab6f4ac4a84d3a8f186#diff-933c557c2433cae9802bf40cf9b4ffbfR364 will see data still as 0 after this call returns.
Otherwise